addons/l10n_eg_edi_eta/models/res_company.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    l10n_eg_client_identifier = fields.Char('ETA Client ID')
+    l10n_eg_client_secret_1 = fields.Char('ETA Secret 1')
```

Comment:
@bakboka This is important.  You will need to do a sudo() to retrieve them.  (makes me think, tests should be done with demo user as well)

```suggestion
    l10n_eg_client_identifier = fields.Char('ETA Client ID', groups='base.group_system')
    l10n_eg_client_secret_1 = fields.Char('ETA Secret 1', groups='base.group_system')
```